.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,102 @@
+name: End-to-end test
```

Comment:
nit: To avoid use of abbreviation could we change this file name to something like `end-to-end`? Really truly a nit.